### DISCUSSION TITLE
Just Special Version of Movie

### DISCUSSION THREAD

**Discussion by saber-rider1** [jellyfin/jellyfin#7024] [Q&A] [answered]
Hey there,

it's possible to store different versions of the same movie in one folder.
You become a choice what version you want to see on the movie page.
But what if you just had the special version without the standard movie?
Then the movie page looks like there's only the standard version.

My folder looks like this:
Movie (YEAR) - Director's Cut.mkv
There is no "Movie (YEAR) - 1080P.mkv

Is it possible to display on the movie page that the only version is the Director's Cut?

Best regards
saber

**Comment by saber-rider1** [comment, marked as answer]
Okay, i solved this to simply edit the title in metadata.
But it's not the best solution, i think.

**Reply by wertercatt** [reply]
I'm not sure another solution would be better, personally. I would assume that if someone had only a single version of a movie, that is the version they prefer to generally watch and consider the 'default' way to watch said movie.

**Reply by saber-rider1** [reply]
Although it may be the standard version for the individual, in reality it is not.
As an example, consider Lord of the Rings. There is the normal movie "Lord of the Rings. And "Lord of the Rings - Extended Version". Two different versions. But I don't think anyone would think of the extended version as the standard version of the movie.
Anyway. I have found a "solution" for me.
